hatchways/team-bluejay
# Become a chef

This has all the commits from [https://github.com/hatchways/team-bluejay/pull/42](url)

To test:
1. run npm start
2. go to localhost:3000
3. open React devtools
4. login with dummy email and password
5. click on "become a chef"
6. supply a meal name, description (optional)
7. Check that the state in BecomeChefButton of the current user has changed to name => Chef, isChef => true
8. Check that the Become a Chef button is no longer there
9. Check the state of meals is no longer an empty array
